lldb/source/Plugins/Process/AIX/NativeProcessAIX.cpp

```diff
@@ -137,6 +137,12 @@ void NativeProcessAIX::Manager::SigchldHandler() {
     auto wait_result = WaitPid();
     if (!wait_result)
       return;
+    lldb::pid_t pid = wait_result->first;
+    WaitStatus status = wait_result->second;
+
+    llvm::any_of(m_processes, [&](NativeProcessAIX *process) {
+      return process->TryHandleWaitStatus(pid, status);
+    });
```

Comment:
How does this relate to kill? It seems unfinished, the return value of any_of is not used.